src/tests/stdlib/make_executable.ab

```diff
@@ -0,0 +1,8 @@
+import { make_executable } from "std"
+main {
+    unsafe $touch /tmp/amber-symbolic$
+    if make_executable("/tmp/amber-symbolic") {
+        echo "created"
+    }
+    unsafe $rm /tmp/amber-symbolic$
```

Comment:
This test should check if the permissions were actually changed

Reply:
Yeah but we don’t know what the function contains. This could theoretically be built with some other command that is more `sh` friendly. But yeah, let’s leave that for now. It’s good enough!